clients/venice-admin-tool/src/main/java/com/linkedin/venice/AdminTool.java

```diff
@@ -1306,7 +1306,7 @@ private static void deleteKafkaTopic(CommandLine cmd) throws Exception {
     long startTime = System.currentTimeMillis();
     String kafkaBootstrapServer = getRequiredArgument(cmd, Arg.KAFKA_BOOTSTRAP_SERVERS);
     Properties properties = loadProperties(cmd, Arg.KAFKA_CONSUMER_CONFIG_FILE);
-    properties.put(KAFKA_BOOTSTRAP_SERVERS, kafkaBootstrapServer);
+    properties.put(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG, kafkaBootstrapServer);
```

Comment:
@haoxu07 -- `org.apache.kafka.clients.consumer.KafkaConsumer` should only appear in `ApacheKafkaConsumerAdapter` and in a rare case in some integration tests. It looks like we missed it at several places. Can you fix it? Thanks!

Reply:
Sure, I can fix it. These `org.apache.kafka.clients.consumer.KafkaConsumer` usages are most from Admin/ChangeCaptureClient.

Reply:
Thanks. Let's also take this opportunity to fix or refactor some other Kafka-related configs.